src/rate.rs

```diff
@@ -589,8 +598,8 @@ impl QuantizerParameters {
         select_ac_qi(quantizer_u, bit_depth).max(1),
         select_ac_qi(quantizer_v, bit_depth).max(1),
       ],
-      lambda: (::std::f64::consts::LN_2 / 6.0)
-        * ((log_target_q as f64) * Q57_SQUARE_EXP_SCALE).exp(),
+      lambda,
+      dist_scale: [1.0, lambda / lambda_u, lambda / lambda_v],
```

Comment:
Hmm, but scalers for YUV are derived from lambdas?! Isn't that essentially equivalent to applying different lambda values when computing rd_cost?